routes/web.php

```diff
@@ -15,10 +17,21 @@
 |
 */
 
+// Route::get('/', function () {
+//     return view('home');
+// })->name('home');
+
 Route::get('/', function () {
-    return view('home');
-})->name('home');
 
+    $randomSong = DB::table('songs')
+        ->inRandomOrder()
+        ->first();
+
+    $imgUrl = Song::find($randomSong->id)
+        ->imageUrl($randomSong->image);
+
+    return view('home', ['song' => $randomSong, 'imgUrl' => $imgUrl]);
+})->name('home');
```

Comment:
Could be outsourced to a controller...

Reply:
Now there is another functionality targeting the database. Both will share the new controller